### PR TITLE
DEVPROD-826 Expose field to determine if a task has test results in cedar

### PR DESCRIPTION
**Pull request by khelif96**
DEVPROD-826

### Description
Exposes a field that determines if a task has test results or not by reading a cached value instead of reaching out to cedar.

The changes are separated out by commit. 
You can omit reviewing 188d2b7 since that just contains gqlgen generated code from updating the dependency.

### CODE REVIEW

File: graphql/task_resolver.go [hunk 1]
```diff
@@ -679,3 +679,13 @@ func (r *taskResolver) VersionMetadata(ctx context.Context, obj *restModel.APITa
...
+
+// !!! WARNING !!!
+// The code below was going to be deleted when updating resolvers. It has been copied here so you have
+// one last chance to move it out of harms way if you want. There are two reasons this happens:
+//   - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//     it when you're done.
+//   - You have helper methods in this file. Move them out to keep these resolver files clean.
+func (r *taskResolver) HasTestResults(ctx context.Context, obj *restModel.APITask) (bool, error) {
+	panic(fmt.Errorf("not implemented: HasTestResults - hasTestResults"))
+}
```
Comment (sophstad) [inline graphql/task_resolver.go:691]: This should probably not be included

**Review by sophstad — CHANGES_REQUESTED** [review]
Would you mind including in the description which files actually contain changes? 😅

File: graphql/task_resolver.go (hunk 1 above)
Comment (khelif96) [reply graphql/task_resolver.go]: > oof good catch.

**Comment by khelif96** [comment]
> Would you mind noting in the description which files actually contain changes? 😅

Updated the description. The changes were separated by commit.

**Review by sophstad — APPROVED** [review]

**Pull request merged**